sites/page-details/includes/footer.html

```diff
@@ -0,0 +1,34 @@
+<footer class="pagedetails{% if include.addContainer %} container{% endif %}">
+	<h2 class="wb-inv">Page details</h2>
+	{%- if page.noReportProblem -%}
+		{%- if page.share -%}
+			<div class="row">
+				<div class="wb-share col-sm-4 col-md-3 col-sm-push-8 col-md-push-9" data-wb-share='{"lnkClass": "btn btn-default btn-block"}'></div>
+				<div class="col-sm-6 col-md-5 col-lg-4 col-sm-pull-4 col-md-pull-3">
+					<dl id="wb-dtmd">
+						<dt>{{ i18nText-dateModified }}</dt>
+						<dd><time property="dateModified">{{ page.dateModified }}</time></dd>
+					</dl>
+				</div>
+			</div>
+		{%- else -%}
+			<dl id="wb-dtmd">
+				<dt>{{ i18nText-dateModified }}</dt>
+				<dd><time property="dateModified">{{ page.dateModified }}</time></dd>
+			</dl>
```

Comment:
Can we have a pattern that avoid to duplicate multiple time the date modified? Like wrapping it as a col-xs-12 component

Reply:
The date modified is now always in a column.